src/useForm.ts

```diff
@@ -120,12 +120,24 @@ const useForm = (
     const handlePayment = async () => {
       const payload = await stripe.createPaymentMethod({
         type: 'card',
-        card: elements.getElement(CardElement)
-        // @TODO: Think whether to pass billing details or not
-        // billing_details: {
-        //   name: formData.name,
-        //   email: formData.email,
-        // }
+        card: elements.getElement(CardElement),
+        billing_details: {
+          ...(formData.name && { name: formData.name }),
+          ...(formData.email && { email: formData.email }),
+          ...(formData.phone && { phone: formData.phone }),
+          ...(formData.address && {
```

Comment:
Yeah true, I'll reformat this and make sure to have it mentioned in the docs in case a specific user wants to catch a customer address as well.

Reply:
Do you have to input an address in order to create a paymentMethod? I know it improves fraud, but is it a requirement?

Reply:
No it's not a requirement, so we shouldn't make it a requirement. It's highly recommended since Radar doesn't work properly without it, but if someone is doing something low risk I think we should support them not needing to collect it